packages/x-components/src/views/home/Home.vue

```diff
@@ -294,7 +302,7 @@
         <template #main-body>
           <!--  Redirection  -->
           <Redirection
-            #default="{ redirection, redirect, abortRedirect, isRedirecting, delayInSeconds }"
+            v-slot="{ redirection, redirect, abortRedirect, isRedirecting, delayInSeconds }"
```

Comment:
I guess it was auto-fixed by eslint, we should keep with `#default`.
```suggestion
            #default="{ redirection, redirect, abortRedirect, isRedirecting, delayInSeconds }"
```